src/window.rs

```diff
@@ -788,6 +788,10 @@ pub enum CursorIcon {
     NwseResize,
     ColResize,
     RowResize,
+
+    /// Represents a custom cursor which uses an image. At the current time, it's only implemented for Windows.
+    #[cfg(target_os = "windows")]
+    Custom(&'static str),
```

Comment:
I think we should mention the accepted formats in this things docs. 

Also, for consistency's sake, we should remove the `#[cfg(target_os = "windows")]`, and instead have other platforms panic with `unimplemented!` when they receive this enum variant.

Reply:
But unimplemented means it looks like it's available ? And then crashes during run..

Reply:
uh i'm gonna wait before fixing the other issues then, personally i think compiling this only under windows made more sense, instead of panic attacks, not sure